lib/ansible/modules/utilities/logic/set_fact.py

```diff
@@ -35,8 +35,9 @@
     required: true
   cacheable:
     description:
-      - This boolean indicates if the facts set will also be added to the
-        fact cache, if fact caching is enabled.
+      - This boolean converts the variable into and actual 'fact' which will also be added to the fact cache, if fact caching is enabled.
+      - Normally this module creates 'host level variables' and has much higher precedence, this option changes the nature and precedence
```

Comment:
Are you able to state what that precedence is in terms of the ones at https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable.

Reply:
well, that list already states 'set_fact' vs 'host fact' are 7 steps away

Reply:
Fair enough, should have actually gone to that link and read it properly